Translated using Weblate (Russian) and Initial Spanish (Argentina) for 1.20.1

marisathewitch opened the pull request:
Translated using Weblate (Russian)

Currently translated at 99,77% (5555 of 5568 strings)

Translated using Weblate Spanish (Argentina)

Currently translated at 1,54% (86 of 5568 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: https://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/

Commit:
Translated using Weblate (Russian)

Currently translated at 99.6% (5548 of 5567 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

f-and committed:
Added translation using Weblate (Spanish (Argentina))

f-and committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 0.3% (17 of 5567 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/es_AR/

f-and committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 1.5% (86 of 5567 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/es_AR/

marisathewitch committed:
Merge branch 'GregTechCEu:1.20.1' into 1.20.1

Commit:
Translated using Weblate (Russian)

Currently translated at 99.6% (5550 of 5568 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 99.7% (5555 of 5568 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 99.7% (5555 of 5568 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 99.7% (5555 of 5568 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

marisathewitch committed:
Merge branch 'GregTechCEu:1.20.1' into 1.20.1

marisathewitch committed:
Merge branch 'GregTechCEu:1.20.1' into 1.20.1

Commit:
Translated using Weblate (Russian)

Currently translated at 99.7% (5555 of 5568 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

screret commented:
please have different languages be in separate PRs, it'll make changes easier to track

f-and committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 1.5% (86 of 5568 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/es_AR/

Commit:
Added translation using Weblate (Portuguese)

Commit:
Added translation using Weblate (Portuguese (Brazil))

Commit:
Deleted translation using Weblate (Portuguese (Portugal))

marisathewitch committed:
Merge branch 'GregTechCEu:1.20.1' into 1.20.1
